code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -430,22 +430,8 @@
 
 		user.mob_light(_range = 3, _color = LIGHT_COLOR_BLOOD_MAGIC, _duration = 0.2 SECONDS)
 
-		var/anti_magic_source = L.anti_magic_check()
-		if(anti_magic_source)
-
-			L.mob_light(_range = 2, _color = LIGHT_COLOR_HOLY_MAGIC, _duration = 10 SECONDS)
-			var/mutable_appearance/forbearance = mutable_appearance('icons/effects/genetics.dmi', "servitude", -MUTATIONS_LAYER)
-			L.add_overlay(forbearance)
-			addtimer(CALLBACK(L, /atom/proc/cut_overlay, forbearance), 100)
-
-			if(istype(anti_magic_source, /obj/item))
-				var/obj/item/ams_object = anti_magic_source
-				target.visible_message(span_warning("[L] starts to glow in a halo of light!"), \
-									   span_userdanger("Your [ams_object.name] begins to glow, emitting a blanket of holy light which surrounds you and protects you from the flash of light!"))
-			else
-				target.visible_message(span_warning("[L] starts to glow in a halo of light!"), \
-									   span_userdanger("A feeling of warmth washes over you, rays of holy light surround your body and protect you from the flash of light!"))
-
+		if(L.can_block_magic(MAGIC_RESISTANCE_HOLY))
```

Comment:
You should definitely revert the changes made to the cult magic that make it unholy, because it makes too many things immune to cult magic.
Cult magic and balance should preferably be addressed in a separate PR, because it seems out of scope for this PR and there's a lot of unintended side-effects from making all of cult magic countered by holy.

Reply:
Agreed.  I will revert the cult changes and open a balance PR for this afterwards.

Reply:
Also I the irony with this spell is that it's only checking for `MAGIC_RESISTANCE`, yet all the text and animation effects are detailed to be for `MAGIC_RESISTANCE_HOLY`.